arch/arm/Kconfig

```diff
@@ -32,4 +32,29 @@ config ARCH_DEFCONFIG
 	string
 	default "arch/arm/defconfig"
 
+choice
+prompt "ARM Architecture logging level"
+default ARCH_ARM_LOG_LEVEL_DBG
+
+config ARCH_ARM_LOG_LEVEL_OFF
```

Comment:
Why do these options need to be specific to ARM and not generic, i.e. for any ARCH?

Reply:
Change done in new PR